src/lib/bootloader/autoyast_converter.rb

```diff
@@ -277,7 +284,11 @@ def export_default(res, default)
 
         DEFAULT_KERNEL_PARAMS_MAPPING.each do |key, method|
           val = default.public_send(method)
-          res[key] = val.serialize unless val.empty?
+          result = val.serialize
+          # do not export resume parameter as it depends on storage which is
+          # not cloned by default and if missing it is proposed (bsc#1187690)
```

Comment:
I would break the sentence in two to make it more understandable.

```
# Do not export the 'resume' parameter as it depends on storage, which is not
# cloned by default. Anyways, 'resume' will be proposed if it's missing (bsc#1187690).
```

Reply:
thanks for note. In the end I found that it is cloned, but it is hard to apply as uuid, id neither path is cloned. Only label one.